uPortal-webapp/src/main/webapp/WEB-INF/jsp/Search/search.jsp

```diff
@@ -18,158 +18,137 @@
-                                                </div>
-                                            </c:forEach>
-                                        </div>
-                                    </div>
-                                </c:forEach>
-                            </div>
-                        </div>
-                    </div>
-                </c:when>
-                    <c:otherwise>
-                        <div class="search-results-empty">
-                            <p><spring:message code="no.results"/></p>
-                        </div>
-                </c:otherwise>
-                </c:choose>
-            </div>
-        </div>
-    </div>
-</div>
 
-<script type="text/javascript" src="${pageContext.request.contextPath}/scripts/search/autosuggestHandler.js"></script>
-<link rel="stylesheet" type="text/css" href="${pageContext.request.contextPath}/css/search/autosuggest.css">
+<template id="search-results-tab-header-template">
+    <a class="mdl-tabs__tab"></a>
+</template>
 
-<%-- Only set prepopulateAutoSuggestUrl if the portlet preference is not empty. --%>
-<c:if test="${not empty portletPreferencesValues['prepopulateAutoSuggestUrl'][0]}">
-    <c:set var="prepopulateAutoSuggestUrl" value="${pageContext.request.contextPath}${portletPreferencesValues['prepopulateAutoSuggestUrl'][0]}"/>
-</c:if>
-
-<script language="javascript" type="text/javascript"><rs:compressJs>
-(function($) {
-
-    $(function() {
-        up.initSearchAuto($, {
-            searchFieldSelector: '#${n}searchInput',
-            prepopulateAutoSuggestUrl: '${prepopulateAutoSuggestUrl}',
-            prepopulateUrlPattern: "${pageContext.request.contextPath}${portletPreferencesValues['prepopulateUrlPattern'][0]}",
-            autoSuggestResultsProcessor: "${portletPreferencesValues['autoSuggestResultsProcessor'][0]}",
-            autoSuggestSearchUrl: '${autocompleteUrl}'
-        });
-        $('#${n}searchResults').tabs();
-        <c:if test="${empty tabKeys}">
-            // Do not display the tabs header when tabKeys is empty
-            $('#${n}searchResults .searchTabsContainer').addClass('hidden');
-        </c:if>
-        $('#${n}searchResults').removeClass('hidden'); // Un-hide the search results now that the tabs are rendered
-    });
+<template id="search-results-tab-panel-template">
+    <div class="mdl-tabs__panel">
+        <ul class="mdl-list"></ul>
+    </div>
+</template>
+
+<template id="search-result-item-template">
+    <li class="mdl-list__item mdl-list__item--three-line">
+        <span class="mdl-list__item-primary-content">
+            <i class="material-icons mdl-list__item-avatar"></i>
+            <span class="up-list-item-title"></span>
+            <span class="mdl-list__item-text-body">
+                <dl></dl>
+            </span>
+        </span>
+        <span class="mdl-list__item-secondary-content">
+            <a class="mdl-list__item-secondary-action" href="#">
```

Comment:
Does this link need ARIA treatment?

Reply:
the link does not. :white_check_mark: 
the icon in the link (line 43) may need some additional annotations.

Reply:
Thanks @ChristianMurphy.  (Which annotations are you thinking of?)

Reply:
I'm going to check if the material icons library by default adds usable text.
Otherwise, I'll add an `aria-label`.